webClient/src/app/editor/code-editor/monaco/hiliters/rexx.ts

```diff
@@ -0,0 +1,184 @@
+    'subword', 'symbol', 'syscpus', 'sysdsn', 'sysvar', 'time', 'trace', 'translate',
+    'trunc', 'userid', 'value', 'verify', 'word', 'wordindex', 'wordlength', 'wordpos',
+    'words', 'xrange', 'x2b', 'x2c', 'x2d'
+  ],
+
+/*******************
+  Unused due to having other pattern matches to do the same in the root scope
+*******************
+
+  "brackets": [
+    ["{", "}"], ["[", "]"], ["(", ")"]
+  ],
+
+  keywords: [
+    'address',
+    'by',
+    'call',
+    'do',
+    'else',
+    'end',
+    'exit',
+    'expose',
+    'for',
+    'forever',
+    'if',
+    'iterate',
+    'interpret',
+    'leave',
+    'nop',
+    'numeric',
+    'otherwise',
+    'parse',
+    'pull',
+    'push',
+    'procedure',
+    'return',
+    'say',
+    'select',
+    'then',
+    'to',
+    'translate',
+    'until',
+    'when',
+    'while',
+
+*/
+  
+  operators: [
+    '+', '-', '*', '/', '%', '//', '**',
+
+    '=', '>', '<', '<<', '>>', '<<=', '>>=',
+    '\\=', '¬=', '/=', '\\<', '¬<', '\\>', '¬>',
+    '\\==', '¬==', '/==', '\\<<', '¬<<', '\\>>', '¬>>',
+    '==', '<=', '>=', 
+
+    '&', '|', '&&', '\\', '¬',
+
+    '<>'
+  ],
+
+  // symbols to see if they match with the operators above
+  symbols:  /[=><&|+\-*\/\%¬\\]+/,
+  
+  tokenizer: {
+    root: [
+      [/\/\*/, 'comment', '@comment' ],
+      
+      //constants
+      [/(['"])[01 ]+\1(b)/, {token: 'constant.rexx'}],
+      [/(['"])[0-9a-fA-F ]+\1(x)/, {token: 'constant.rexx'}],
+
+      // strings
+      [/"([^"\\]|\\.)*$/, 'string.invalid'], // non-teminated string
+			[/'([^'\\]|\\.)*$/, 'string.invalid'], // non-teminated string
+			[/"/, 'string', '@doublestring'],
+			[/'/, 'string', '@singlestring'],
+
+      //procedure
+      //entity.name.function.rexx
+      [/\b[A-Za-z@#$!?_][A-Za-z@#$!?_0-9]*:/, {token: 'constructor'}],
+
+      // numbers
+      [/([0-9]+(\.)?[0-9]*(e[-+]?[0-9]+)?|[0-9]*(\.)?[0-9]+)(e[-+]?[0-9]+)?\b/, 'number'],
+
+      //other constants
+//disabled because the coloring was overbearing
+//      [/[0-9\.][A-Za-z0-9@#$¢.!?_]*/, {token: 'constant.other.rexx'}],
+
+      //operators
+//disabled because theoperator section above root is used instead
+//      [/([\+-/*%&amp;|()¬\\=&lt;&gt;])/, {token: 'operators.rexx'}],
+
+      //call function calls
+      [/(call)(\s+[A-Za-z@#$!?_0-9]+)/, ['keyword', { cases: { '@functions': 'builtin-fcall', '@default': 'fcall'}}]],
+
+      //keywords
+      [/\b(do|forever|while|until|to|by|for|end|exit|if|then|else|iterate|leave|nop|return|select|when|otherwise|call(\s+(off|on)\s+(error|failure(\s+name)?|halt))?|signal(\s+(off|on)\s+(error|failure(\s+name)?|halt|novalue|syntax))|address\s+\w+|arg|drop|interpret|numeric\s+(digits|form(\s+(scientific|engineering|value))?|fuzz)|options|parse(\s+upper)?\s+(external|numeric|source|value|var|version)?|with|procedure(\s+expose)?|pull|push|queue|say|trace\s+\w+|upper)\b(?!\.)/, {token: 'keyword'}],
```

Comment:
I am curious why do you need `address\s+\w+` and `trace\s+\w+`?
Editor highlights `TRACE` and `ADDRESS` like this:
![image](https://user-images.githubusercontent.com/34669449/102179889-c3d81900-3ec9-11eb-973c-faafa78688ae.png)
Let's compare with ISPF editor:
![image](https://user-images.githubusercontent.com/34669449/102179993-f4b84e00-3ec9-11eb-8127-c6a628414181.png)

If I remove `s+\w+` in your code then Editor highlights `TRACE` and `ADDRESS` like this:
![image](https://user-images.githubusercontent.com/34669449/102179552-2846a880-3ec9-11eb-9ad6-818bfb2b3859.png)